src/surfaces/actionBar/actionBar.scss

```diff
@@ -347,41 +497,53 @@ $backwards-step4-duration: $standard-fifth-timing;
 
 @media only screen and (min-width: 768px) {
     .ui.drawer.drawer-has_action_bar {
+
         .drawer-container-inner,
         .drawer--wing-container-inner {
             padding-top: $drawer-action-bar-height;
         }
     }
+
     .ui.drawer.drawer-has_title_bar.drawer-has_action_bar {
+
         .drawer-container-inner,
         .drawer--wing-container-inner {
             padding-top: $drawer-title-bar-height + $drawer-action-bar-height;
+
             .ui.drawer--action_bar {
                 top: $drawer-title-bar-height;
             }
         }
     }
+
     .ui.drawer.drawer-has_title_bar.drawer-has_navigation.drawer-has_action_bar {
+
         .drawer-container-inner,
         .drawer--wing-container-inner {
             padding-top: $drawer-title-bar-height + $drawer-navigation-height + $drawer-action-bar-height;
+
             .ui.drawer--action_bar {
                 top: $drawer-title-bar-height + $drawer-navigation-height;
             }
         }
     }
+
     .ui {
-        &.drawer--action_bar, &.page--action_bar {
+
+        &.drawer--action_bar,
+        &.page--action_bar {
             padding: 0 22px;
         }
+
         &.page--action_bar {
             left: 250px;
             min-height: 70px;
             padding: 0 22px;
             top: 70px;
         }
+
         &.drawer--action_bar {
             height: $drawer-action-bar-height;
         }
     }
-}
+}
```

Comment:
Can we retain blank line at the end please? =)

Reply:
Ahh, auto-format on save did that 😆 . Fixed now.

Reply:
Auto-formaaaat!!! 😬 🤣